netiquette/README.md

```diff
@@ -0,0 +1,63 @@
+# BcnEng's slack netiquette
+
+Welcome to BcnEng's slack netiquette. If you don't agree with some guideline or whish to propose an improvement, feel free to open a pull request.
+
+## Definition
+
+From [wikipedia](https://en.wikipedia.org/wiki/Etiquette_in_technology#Netiquette):
+
+> Netiquette [...] is a set of social conventions that facilitate interaction over networks [...]
+
+## Conventions
+
+Here you will find a set of basic conventions that we find useful.
+
+### behaviour
+
+Be respectful with your colleagues and follow the [Code of Conduct](https://github.com/bcneng/bcneng/tree/master/coc) rules.
```

Comment:
I don't think `CoC` lists _rules_. Sounds a bit too string, doesn't it ?

Reply:
From Wikipedia: 

> A code of conduct is a set of rules outlining the social norms,religious rules and responsibilities of, and or proper practices for, an individual, ...